examples/ic_api/test/test.ts

```diff
@@ -10,6 +10,51 @@ const ic_api_canister = createActor('rrkah-fqaaa-aaaaa-aaaaq-cai', {
 
 const tests: Test[] = [
     ...cleanDeploy('ic_api'),
+    {
+        name: 'arg_data_raw',
+        test: async () => {
+            const blobString = 'Surprise!';
+            const blob = blobString.split('').map((char) => char.charCodeAt(0));
+            const int = 127;
+            const bool = true;
+            const string = 'test';
+            const candidString = `(blob "${blobString}", ${int} : int8, ${bool}, "${string}")`;
+
+            const resultBytes = await ic_api_canister.arg_data_raw(
+                blob,
+                int,
+                bool,
+                string
+            );
+            const result = candidDecode(resultBytes);
+
+            return {
+                ok: result === candidString
+            };
+        }
+    },
+    {
+        name: 'arg_data_raw_size',
+        test: async () => {
+            const blobString = 'Surprise!';
+            const blob = blobString.split('').map((char) => char.charCodeAt(0));
+            const int = 127;
+            const bool = true;
+            const string = 'test';
+            const candidString = `(blob "${blobString}", ${int} : int8, ${bool}, "${string}")`;
+
+            const result = await ic_api_canister.arg_data_raw_size(
+                blob,
+                int,
+                bool,
+                string
+            );
+
+            return {
+                ok: result === 29n
```

Comment:
I think we should get the length as part of the test instead of hard-coding it, doing a call to ic_api_canister.arg_data and getting the length

Reply:
Agreed. I've fixed this. Thanks for the help!